uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/zones/OrchestratorZoneControllerMockMvcTests.java

```diff
@@ -165,7 +164,7 @@ void testGetZone_nameRequiredError(String url) throws Exception {
     void testGetZone_nameEmptyError(String url) throws Exception {
         performMockMvcCallAndAssertError(get(url), status().isBadRequest(),
                                          JsonUtils.writeValueAsString(
-                                             new OrchestratorErrorResponse("getZone.name: must not be empty")),
+                                             new OrchestratorErrorResponse("name: must be specified")),
```

Comment:
Not sure about this error message "name: must be specified", how this is coming? from api response or just test data but all error messages should follow same format. Include or remove ":" from all error messages.

Reply:
Done . Thanks!

Retrun error response as {
"message": "name must be specified"
} for both get and delete zone.